wolfssl/ssl.h

```diff
@@ -3206,6 +3206,45 @@ WOLFSSL_API void  wolfSSL_CTX_SetRsaDecCb(WOLFSSL_CTX*, CallbackRsaDec);
 WOLFSSL_API void  wolfSSL_SetRsaDecCtx(WOLFSSL* ssl, void *ctx);
 WOLFSSL_API void* wolfSSL_GetRsaDecCtx(WOLFSSL* ssl);
 #endif
+
+/* Protocol Callback */
+typedef int (*CallbackGenMasterSecret)(WOLFSSL* ssl, void* ctx);
+WOLFSSL_API void  wolfSSL_CTX_SetGenMasterSecretCb(WOLFSSL_CTX*, CallbackGenMasterSecret);
+WOLFSSL_API void  wolfSSL_SetGenMasterSecretCtx(WOLFSSL* ssl, void *ctx);
+WOLFSSL_API void* wolfSSL_GetGenMasterSecretCtx(WOLFSSL* ssl);
+
+typedef int (*CallbackGenPreMaster)(WOLFSSL* ssl, byte *premaster,
+                                                   word32 preSz, void* ctx);
+WOLFSSL_API void  wolfSSL_CTX_SetGenPreMasterCb(WOLFSSL_CTX*, CallbackGenPreMaster);
+WOLFSSL_API void  wolfSSL_SetGenPreMasterCtx(WOLFSSL* ssl, void *ctx);
+WOLFSSL_API void* wolfSSL_GetGenPreMasterCtx(WOLFSSL* ssl);
+
+typedef int (*CallbackGenSessionKey)(WOLFSSL* ssl, void* ctx);
+WOLFSSL_API void  wolfSSL_CTX_SetGenSesssionKeyCb(WOLFSSL_CTX*, CallbackGenSessionKey);
+WOLFSSL_API void  wolfSSL_SetGenSesssionKeyCtx(WOLFSSL* ssl, void *ctx);
+WOLFSSL_API void* wolfSSL_GetGenSesssionKeyCtx(WOLFSSL* ssl);
+
+typedef int (*CallbackSetKeys)(WOLFSSL* ssl, void* ctx);
+WOLFSSL_API void  wolfSSL_CTX_SetSetKeysCb(WOLFSSL_CTX*, CallbackSetKeys);
```

Comment:
If we change the callback to `EncryptKeysCb`, these API would change names to `wolfSSL_CTX_SetEncryptKeysCb`.  Applies to all `SetKeys` naming.

Reply:
Changed all SetKeys to EncryptKeys